onnx_xla/translations/translate_batch_normalization.cc

```diff
@@ -7,9 +7,14 @@ namespace onnx_xla {
 onnxStatus translateBatchNormalization(const Node& n,
                                        XlaBuilder& builder,
                                        ValueOpMap& valueToOp) {
-  if (n.outputs().size() > 1 || n.hasAttribute(kmomentum) ||
-      n.hasAttribute(Symbol("spatial"))) {  // TODO: ENFORCE
-    throw std::runtime_error("Only test mode of BatchNormalization supported");
+  // If requires output of train mode, throw
+  if (n.outputs().size() > 1) {
```

Comment:
1. doesn't seem to need this `if`
2. keeping the attribute check is better IMO

Reply:
Removing unneeded if and bringing back attribute checks